### PR TITLE
Filled out necessary metadata for `crates.io`

### PR DESCRIPTION
**Pull request by Swandog**
Made my best guess with the descriptions. Some of them I pulled from READMEs, other from commit messages.

### CODE REVIEW

**Review by schmidmt — APPROVED** [review]
I think these are okay. Maybe @BaxterEaves has a stronger opinion.

**Comment by Swandog** [comment]
FWIW I think these can be updated after the fact if we want

File: lace/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,9 @@ edition = "2021"
 exclude = ["tests/*", "resources/test/*", "target/*"]
 rust-version = "1.58.1"
 license = "SSPL-1.0"
+homepage = "https://www.lace.dev/"
+repository = "https://github.com/promised-ai/lace"
+description = "A probabilistic cross-categorization engine written in rust with an optional interface to python"
```
Comment (BaxterEaves) [inline lace/Cargo.toml:12]: Just "A probabilistic cross-categorization engine" is good.

**Review by BaxterEaves — COMMENTED** [review]
One change

**Pull request merged**